client/app/scripts/stores/app-store.js

```diff
@@ -44,6 +44,7 @@ function makeNode(node) {
 
 // Initial values
 
+let changingTopologies = false;
```

Comment:
Thinking out loud: instead of not-showing-the-warning based on this flag, which I think could be a pain to maintain, could you use the stats from the regular fetch of /api/topology to indicate if the warning should be shown?